src/live-region/__tests__/live-region.test.tsx

```diff
@@ -9,7 +9,7 @@ import styles from '../../../lib/components/live-region/test-classes/styles.css.
 
 const renderLiveRegion = async (jsx: React.ReactElement) => {
   const { container } = render(jsx);
-  await waitFor(() => expect(document.querySelector('[aria-live=polite]')));
+  await waitFor(() => expect(document.querySelector('[aria-live]')).toBeTruthy());
```

Comment:
Remove `polite` here as we have tests that render an assertive live region.
https://github.com/cloudscape-design/components/blob/main/src/live-region/__tests__/live-region.test.tsx#L90